docs/src/app/shared/theme-picker/theme-picker.ts

```diff
@@ -85,19 +85,18 @@ export class ThemePicker implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
-    this._queryParamSubscription = this._activatedRoute.queryParamMap
-      .pipe(map((params: ParamMap) => params.get('theme')))
+    this._activatedRoute.queryParamMap
+      .pipe(
+        map((params: ParamMap) => params.get('theme')),
+        takeUntilDestroyed(this._destroyRef),
```

Comment:
nit: We can run this in the constructor context and drop that destroyRef.
```suggestion
        takeUntilDestroyed(),
```